### ISSUE TITLE
Very high tick rate if Mekanism cables are connected at the same time-use with RF and EU machines.

### ISSUE THREAD

**Issue by Steelpower900** [aidancbrady/Mekanism#2344]
in Mekanism.cfg Blacklist

RF_on / EU_on
![rf_eu-power on](https://cloud.githubusercontent.com/assets/8689620/6800979/f6dad284-d223-11e4-821f-a2099afb6409.PNG)

RF_off / EU_on
![rf_off-eu_on](https://cloud.githubusercontent.com/assets/8689620/6801028/79af57d4-d224-11e4-94b3-7bf312888373.PNG)

RF_On - EU_Off
![rf_on-eu_off](https://cloud.githubusercontent.com/assets/8689620/6801043/a945adcc-d224-11e4-9c2b-9331fd6c95c2.PNG)

RF_off - EU_off
![rf_eu-power off](https://cloud.githubusercontent.com/assets/8689620/6801048/c817163c-d224-11e4-8099-5c86986a73f9.PNG)

Mekanism 1.7.10-8.0.1.198

**Comment by DJDayta** [comment]
we had something above 5000 mekanism cables in use and it turne out to be a strange kind of lag. with the help of heatmaps and otis we solved the mystery a little bit that is. for example if player a was in his base A everything was running smooth for him. as soon as he leaves his base A and the cables are now in chunks that are not loaded by the player already lagging started to mount up to 40 ms befor disconnecting the player. a solution was if when you travled around you just had to logout wait for the chunks temporarily loading to shut down completly and relog to the server after making sure no chunks  were loaded. 
we came to the plan to try to avoid any energy related things from mekanism that is the energy box the cable and everything that generates power or has the capacity to pass it along. windmill for example have also a geat impact concidering alternatives are so far below. for comparison 2 windmills are about the same amount as one cow on the server. you might say ok there are hundret of cows thats true but there are close to 20000 windmills running and people were just playing for a week we have some insane guys who are used to go totaly balistic on theese things. 


anyway long story short 
here some examples





we used mekanism 7 as well as 8

**Comment by aidancbrady** [comment]
Is this still occurring with the latest Mekanism builds?

**Comment by Steelpower900** [comment]
Unfortunately, only a slight improvement from 500ms to 300ms.

Maybe you help my sampler further from my server,-

RF_on / EU_on
http://www.mediafire.com/download/f18st7cf9j9iszz/Mekanism-1.7.10-8.0.3.207.nps
and
http://www.mediafire.com/download/7fso5nvjueqbnsi/Mekanism-1.7.10-8.0.3.212.nps

**Comment by Steelpower900** [comment]
Unfortunately, only a slight improvement from 500ms to 300ms.

Maybe you help my sampler further from my server,-

RF_on / EU_on
http://www.mediafire.com/download/f18st7cf9j9iszz/Mekanism-1.7.10-8.0.3.207.nps
and
http://www.mediafire.com/download/7fso5nvjueqbnsi/Mekanism-1.7.10-8.0.3.212.nps

**Comment by aidancbrady** [comment]
@Steelpower900, you don't know how helpful those snapshots have been. I think I found the issue.

**Comment by Steelpower900** [comment]
@aidancbrady I would like to test the build on my root server :)

**Comment by Steelpower900** [comment]
ok a little update this connection can here the Tick Time sharp rises higher the decrease in energy is the worse the effect. 

![2015-03-30_14 54 38](https://cloud.githubusercontent.com/assets/8689620/6896614/776cc862-d6ed-11e4-8aac-4dc0519913b4.png)

**Comment by aidancbrady** [comment]
@Steelpower900, here's the URL: https://dl.dropboxusercontent.com/u/90411166/Mekanism-1.7.10-8.0.3.615.jar

Let me know how it works out.

**Comment by Steelpower900** [comment]
Unfortunately, the problem is still not entirely clear but jetz is the refresh rate decreased from 300ms to 100ms.

Here again a current sampler,-

http://www.mediafire.com/download/y7g8f8tozfwpp35/Mekanism-1.7.10-8.0.3.615.nps

but you seem to be on the right track :)

**Comment by aidancbrady** [comment]
Thanks for reporting back!
After taking a quick look at the snapshot, the lag you're experiencing actually isn't a result of Mekanism - it's a result of a chain that goes through TE conduits, TE tesseracts, and then an AE system.  I can provide more details for you if necessary, but my work here is definitely done :)

Below is a chart of the CPU stress-related calls you experienced - as you can see, very little is actually used by Mekanism.
![usage]
(http://puu.sh/gVMty/177003e3f8.png)

**Comment by Steelpower900** [comment]
Ok Aidancbrady thanks for your help I'll contributing here at TeamCoFH link maybe the guys there can still improve on what TE :)
